osgp/shared/osgp-dto/src/main/java/org/opensmartgridplatform/dto/valueobjects/smartmetering/CircuitSwitchedStatusDto.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2020 Alliander N.V.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ */
```

Comment:
Wrong year, and please use non-javadoc comment style with the license header to avoid the `<p>`-tags added by the formatter in javadoc paragraphs.